plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src/org/wso2/developerstudio/eclipse/gmf/esb/provider/FailoverEndPointItemProvider.java

```diff
@@ -61,11 +61,34 @@ public FailoverEndPointItemProvider(AdapterFactory adapterFactory) {
 		
 		super.getPropertyDescriptors(object);
 		addPropertiesPropertyDescriptor(object);
+		addBuildMessagePropertyDescriptor(object);
 	
 		return itemPropertyDescriptors;
 	}
 
 	/**
+	 * This adds a property descriptor for the Build Message feature.
+	 * <!-- begin-user-doc -->
+	 * <!-- end-user-doc -->
+	 * @generated NOT
+	 */
```

Comment:
Shall we elaborate the javaDocs by explaining the usage of parameters ?

Reply:
Fixed

Reply:
Fixed